pylint/__pkginfo__.py

```diff
@@ -13,3 +13,7 @@
         __version__ += f"a{dev_version}"
     else:
         __version__ += f".dev{dev_version}"
+
+
+# Kept for compatibility reason, see https://github.com/PyCQA/pylint/issues/4399
+numversion = __version__.split(".")
```

Comment:
In <2.8 is was a `tuple`, not a `list`, see https://github.com/PyCQA/pylint/commit/fbcb27acb3431cea846bbfb997f411e51417d1bb#diff-42df8bb8e2d7b0a18f0174dcb5f3c60b1cbca5a394c3c7f8c9b33c99fb6cda99L34

`list` causes the following `TypeError` in pylint-django (PyCQA/pylint-django#323)

```python
LOAD_CONFIGURATION_SUPPORTED = pylint.__pkginfo__.numversion >= (2, 3)
TypeError: '>=' not supported between instances of 'list' and 'tuple'
```